wordpress.yaml

```diff
@@ -1,101 +1,243 @@
 name: WordPress
-description: 'A Model Context Protocol (MCP) server that integrates with a self-hosted
-  WordPress site (wordpress.com not supported).
-
-
+description: |
+  A Model Context Protocol (MCP) server that integrates with a self-hosted WordPress site (wordpress.com not supported).
+  
   ## Features
-
-  - **WordPress Feature API**: Standardized adapter providing streamlined access to
-  core WordPress functionality
-
-  - **REST API CRUD Tools**: Flexible generic tools for interacting with any WordPress
-  REST API endpoint, including APIs from plugins.
-
-
-  ## What you''ll need to connect
-
-
-  ### Install plugin to your WordPress site
-
-  - Download `wordpress-mcp.zip` from the [plugin releases page](https://github.com/Automattic/wordpress-mcp/releases)
-
-  - Upload the plugin to your WordPress site in the `/wp-content/plugins/wordpress-mcp`
-  directory
-
-  - Activate the plugin.
-
-  - Navigate to Settings > MCP, enable the MCP functionality, and configure other
-  settings as needed.
-
-
-  ### have your credentials ready
-
+  - **Posts**: Manage blog posts
+  - **Categories and Tags**: Create, read, update, delete categories and tags
+  - **Media**: Manage the uploaded media files
+  
+  ## What you'll need to connect
+  
+  **Required:**
   - **WordPress Site URL**
-
   - **WordPress username**
+  - **WordPress app password**: this is different from your user password, and can be generated in the settings
 
-  - **WordPress app password**: The application password that you generated in your
-  WordPress settings. Note -- This is different from your user password.
-
-  '
 metadata:
   categories: Blogging, SaaS & API Integrations
 icon: https://img.icons8.com/?size=100&id=13664&format=png&color=000000
-repoURL: https://github.com/Automattic/wordpress-mcp
+repoURL: https://github.com/obot-platform/wordpress-mcp-server
+
+toolPreview:
+  - name: create_category
```

Comment:
nit: outdent these and other arrays one level. Otherwise, the next run of the automation will have a bunch of whitespace changes. See https://github.com/obot-platform/mcp-catalog/pull/85/files as an example.

Reply:
I think most of the whitespace changes have been fixed, but the weird hard line-wrapping still gets removed.

Reply:
updated.